libexec/python/shub/api.py

```diff
@@ -114,15 +114,15 @@ def download_image(manifest,download_folder=None,extract=True):
     '''    
     image_file = get_image_name(manifest)
 
-    print("Found image %s:%s" %(manifest['name'],manifest['branch']))
-    print("Downloading image... %s" %(image_file))
+    sys.stderr.write("Found image %s:%s\n" %(manifest['name'],manifest['branch']))
+    sys.stderr.write("Downloading image... %s\n" %(image_file))
 
     if download_folder != None:
         image_file = "%s/%s" %(download_folder,image_file)
     url = manifest['image']
     image_file = api_get(url,stream=image_file)
     if extract == True:
-        print("Decompressing", image_file)
+        sys.stderr.write("Decompressing %s\n" %(image_file))
         os.system('gzip -d -f %s' %(image_file))
```

Comment:
Please see the logger that we have, which is in sync with the SINGULARITY_MESSAGELEVEL

https://github.com/singularityware/singularity/blob/master/libexec/python/logman.py